src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    class="searchbar"
+    data-qa="big"
+    >
+      <img src="images/Search.svg"
+      alt="search"
+      class="searchIcon"
+      >
```

Comment:
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      class="searchbar"
      data-qa="big"
    >
      <img 
        src="images/Search.svg"
        alt="search"
        class="searchIcon"
      >
```
![image](https://user-images.githubusercontent.com/96409563/218988897-f7e4345e-b823-4453-b1f3-c9bf13f58e7e.png)
Please fix all cases